### PR TITLE
chore: Bump snapbuild packages

### PR DESCRIPTION
**Pull request by jennifer-shehane**
### Additional details

This bumps the `@cypress/snapbuild-*` packages from 1.0.2 to 1.0.3.

This update should have no effect as per [this Slack conversation](https://cypressio.slack.com/archives/D02NG0KQNET/p1699041198387049). Just want to get them off the list of deps that keep showing up on [this list](https://github.com/cypress-io/cypress/issues/3777) that need update. 

Not sure if I need to run some v8 snapshot tasks for this update or not.

### Steps to test

Tests should pass

### How has the user experience changed?

Should be no difference.

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

File: tooling/v8-snapshot/cache/linux/snapshot-meta.json [hunk 1]
```diff
@@ -604,6 +604,8 @@
     "./node_modules/xdg-basedir/index.js",
     "./node_modules/xml2js/lib/xml2js.js",
     "./packages/config/index.js",
+    "./packages/config/src/browser.ts",
+    "./packages/config/src/index.ts",
```
Comment (jennifer-shehane) [inline tooling/v8-snapshot/cache/linux/snapshot-meta.json:608]: Looks like these were just reordered....

**Comment by jennifer-shehane** [comment]
@ryanthemanuel I'm thinking this change does have an effect on things 😅

**Comment by ryanthemanuel** [comment]
@jennifer-shehane, the launchpad failures were weird. There was a busy port: https://cloud.cypress.io/projects/ypt4pf/runs/56583/overview/2a8cb34e-979c-4d00-a494-b2f14d8d962a/replay?roarHideRunsWithDiffGroupsAndTags=1&ts=1723741128942.5999&att=1

is it possible that was flake? I'm not sure I've seen it before, but everything now seems to be passing.

**Comment by jennifer-shehane** [comment]
@ryanthemanuel Alright welp, the tests were clearly lying.

**Review by ryanthemanuel — APPROVED** [review]

**Review by cacieprins — APPROVED** [review]

**Pull request merged**